vpr/src/place/net_cost_handler.cpp

```diff
@@ -55,14 +55,10 @@ static vtr::NdMatrix<float, 2> chanx_place_cost_fac({0, 0}); // [0...device_ctx.
 static vtr::NdMatrix<float, 2> chany_place_cost_fac({0, 0}); // [0...device_ctx.grid.height()-2]
 
 /**
- * @brief Cost of a net, and a temporary cost of a net used during move assessment. 
+ * @brief For the first two element in the struct: Cost of a net, and a temporary cost of a net used during move assessment. 
```

Comment:
You should use the member names (net_cost etc.) instead of the order in the struct, in case a member is added or deleted or reordered in the future. It also makes the comment more self-contained.

Reply:
Also comment that there is one entry per cluster level net [0...cluster_ctx.clb_nlist.nets().size()-1] for each of these vectors. (Mention this early in the comment).